lib/server/routers/runs.router.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     children: [
         {
             path: '/runs',
-            args: { public: true },
```

Comment:
Probably best is to simply put `args: {public: true}` on top and remove the sub-ones no?

Reply:
You mean `public: false` I guess?
This is the default behaviour of the server in any way, but I let it as false to be explicit

Reply:
Yes public false.
TBH I would rather delete it from everywhere if it's the default, to me it looks like dead code.
But my comment was: removing it from children and putting it only at the top

Reply:
I was not sure if you want to keep the convention as it was from the others.
Done, I removed all and let the default `visible` to be applied by the server.